pages/_app.tsx

```diff
@@ -21,6 +23,28 @@ const globalStyles = globalCss({
 
 function MyApp({ Component, pageProps }: AppProps) {
   globalStyles();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (router.pathname !== "/blocked") {
+      const script = document.createElement("script");
+      script.src = "https://app.termly.io/embed.min.js";
```

Comment:
What's this and why do we need it?

Reply:
Hi, @OisinKyne I just saw that comment, and for some reason didn’t get that notification, this is actually the opposite, I’m not adding the termly script when it goes to the blocked page. that means, that the page it will not raise the cookies banner

Reply:
Okay fair enough. If this was dvl I'd maybe push for not using a third party tracker, but for site its fine, its just a landing page for people that bounce off.